MSAL/test/unit/native_auth/mock/MSALNativeAuthNetworkMocks.swift

```diff
@@ -205,7 +224,7 @@ class MSALNativeAuthSignInRequestProviderMock: MSALNativeAuthSignInRequestProvid
     
     func challenge(parameters: MSAL.MSALNativeAuthSignInChallengeRequestParameters, context: MSIDRequestContext) throws -> MSIDHttpRequest {
         checkContext(context)
-        if let expectedCredentialToken = expectedCredentialToken {
+        if let expectedCredentialToken = expectedContinuationToken {
```

Comment:
If we get rid of expectedCredentialToken in the variables above, should we just say:
```
if let expectedCredentialToken {
   XCTAssertEqual(expectedContinuationToken, parameters.continuationToken)
}
```

Reply:
Thanks, I updated the whole class with this suggestion